cmd/platform/server.go

```diff
@@ -130,9 +130,11 @@ func runServer(configFileLocation string) {
 	}
 
 	if a.Elasticsearch != nil {
-		if err := a.Elasticsearch.Start(); err != nil {
-			l4g.Error(err.Error())
-		}
+		go func() {
```

Comment:
Should probably use `a.Go(func(){` to ensure that the function doesn't return while this goroutine is still executing.